### DISCUSSION TITLE
Markdoc Language Server does not respond to config

### DISCUSSION THREAD

**Discussion by louiss0** [markdoc/markdoc#428] [Q&A] [answered]
### What happened?

I was trying to set up the config with the Markdoc language server but I got nothing.

I put the `markdoc.config.json` file in the root of the project and set everything to this.

```json
[
  {
    "id": "my-site",
    "path": "src/content/docs",
    "schema": {
      "path": "markdoc.config.mjs",
      "type": "node",
      "property": "default",
      "watch": true
    },
    "routing": {
      "frontmatter": "route"
    }
  }
]

``` 

Then I installed a library called [Markdoc HTML Tag Schemas](https://markdoc-html-tag-schemas-docs.onrender.com) 

and configured the `markdoc.config.mjs` file like this. 

```mjs
import { defineMarkdocConfig, component } from "@astrojs/markdoc/config";
import { markdocHTMLTagSchemas } from "markdoc-html-tag-schemas";

export default defineMarkdocConfig({
  extends: [markdocHTMLTagSchemas()],
  tags: {
    aside: {
      render: component("./src/components/Aside.astro"),
      attributes: {
        type: { type: String },
        title: { type: String },
      },
    },
  },
});

```

I got no autocomplete at all. 

### To reproduce

All you have to do is just clone the repo in order to reproduce the steps.

### Version

0.3.0

### Additional context

I'm using the `@astrojs/markdoc` extension to run markdoc. It uses `ts,js,mjs` to run markdoc. It also has a property called `extend:`

It's a prop that expects an array of objects that are used to configure markdoc. I decided to use this property to configure Markdoc.
The config also includes a tag that hard coded. The extension that i used comes for [Markdoc Html Tag Schemas](https://markdoc-html-tag-schemas-docs.onrender.com). It's a library that contains many schemas that are used you can also import schemas from the library as well 

```js
 import { ul } from "markdoc-html-tag-schemas"

export default {

tags:{
ul
}
}
 
```

**Comment by rpaul-stripe** [comment, marked as answer]
The "type" property under "schema" in the config should be set to "esm" instead of "node" for ES modules (files with the ".mjs" extension). The "node" type is for Node-style "module.exports" modules that need to be imported with `require`. Sorry if that wasn't clear. I will take a look at the error handling and make it display an actual error message when it fails to import in this way.

If it is still not working as expected after changing the configuration, feel free to reopen and we can troubleshoot further.

**Reply by louiss0** [reply]
Ok now it works but the extends keyword is not supported I'll talk to ben about this. Thank you. I hope we can get auto-close support for paired tags and maybe even auto editing as well.

**Reply by louiss0** [reply]
I got it to work but now there is another problem I think it can be discussed here instead of having to open another issue.
